samples/Console/Console.csproj

```diff
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
It seems this prevents running the console app, as netstandard2.0 is not a runnable target framework.
What do you think would be best here ? Maybe multiple target frameworks ?

Reply:
Ah, yes, I missed that. The target framework versioning throws me off every time I have to deal with it as well.

`net5.0` will compile, but with warnings, since .NET 5.0 is no longer supported.

`net6.0` is the current LTS version, so that would probably be the best choice. If someone is still running .NET Framework 4.8, they can change the target and it will compile for them.

You could multi-target `net6.0; net48`, but then you have to add a `--framework` flag when you compile to specify which one you want.